Data/include/Poco/Data/SimpleRowFormatter.h

```diff
@@ -109,13 +109,10 @@ inline std::streamsize SimpleRowFormatter::getSpacing() const
 
 namespace std
 {
-	// Note: for an unknown reason, clang refuses to compile this function as noexcept
+	// Note: for an unknown reason, clang and gcc-15 refuses to compile this function as noexcept
 	template<>
 	inline void swap<Poco::Data::SimpleRowFormatter>(Poco::Data::SimpleRowFormatter& s1,
 		Poco::Data::SimpleRowFormatter& s2)
-#ifndef POCO_COMPILER_CLANG
-		noexcept
```

Comment:
Proper way is to use:
```
noexcept(std::is_nothrow_swappable_v<Poco::Data::SimpleRowFormatter>)
```

as specified here:  https://en.cppreference.com/w/cpp/algorithm/swap

Reply:
Switched to your variant. Still builds for me on `gcc-15` and `gcc-13`. Thank you!